tests/typecheck/managers/querysets/test_as_manager.yml

```diff
@@ -310,7 +310,7 @@
 -   case: handles_type_collision_with_from_queryset
     main: |
         from myapp.models import MyModel, FromQuerySet
-        reveal_type(FromQuerySet)  # N: Revealed type is "def [_T <: django.db.models.base.Model] () -> myapp.models.ManagerFromModelQuerySet[_T`1]"
+        reveal_type(FromQuerySet)  # N: Revealed type is "def () -> myapp.models.ManagerFromModelQuerySet[_T`1]"
```

Comment:
`T` will be unbound here

Reply:
Hm, yeah, gotta look in to that. But I'm not so sure there should be any arguments here. It might be, if we implicitly copy the manager argument to the new subclass created. It's all a bit blurry, since it needs to align with the model argument of the queryset etc.